tests/conftest.py

```diff
@@ -991,11 +993,34 @@ def generate_greedy_logprobs(
                                         videos=videos,
                                         **kwargs)
 
+    def generate_prompt_perplexity(self, prompts: list[str]) -> list[float]:
+        outputs = self.generate_greedy_logprobs(prompts,
+                                                max_tokens=1,
+                                                num_logprobs=None,
```

Comment:
What does None mean here?

Reply:
`num_logprobs =None` means do not produce logprobs for decoded tokens, whereas `num_prompt_logprobs=0` means produce one logprob for each prefill (prompt) token